app/feature.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	EnableExportFeatureFlag = &cli.BoolFlag{
+		Name:     "enable-export",
```

Comment:
[Nit] Let's just call this `toggle-export` and it will just invert whatever value is already in the config map. Otherwise, the way this is, if we had multiple feature flags, a user has to remember to add the flags for the existing ones already enabled, which is confusing